quickstarts/loadmill/config.yml

```diff
@@ -0,0 +1,42 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: loadmill
+
+# Displayed in the UI (required)
+title: Loadmill Tests Results
+
+# Long-form description of the quickstart (required)
+description: |
+  Monitor your Loadmill's test results.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Monitor your Loadmill's test results
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Loadmill L.T.D
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
```

Comment:
Hello! If you want your quickstart to appear under a specific category please use the keywords here in the query. They map to specific categories. 

https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A

Reply:
I added `network` though its not the best category. Is there an option to add a testing category?